Add hacktoberfest program. Remove year from program titles. Delete gs…

jesuslerma opened the pull request:
**Description**
I added a new program for hacktoberfest 2020 participation.
This PR fixes #829 

Hi @prateeksahu and @chandrashritii can u please review this entry?
**Notes for Reviewers**


**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.
 

<!--
Thank you for contributing to Layer5 projects! 

Contributing Conventions:

1. Include descriptive PR titles with [<component-name>] prepended.
2. Build and test your changes before submitting a PR. 
3. Sign your commits

By following the community's contribution conventions upfront, the review process will 
be accelerated and your PR merged more quickly.
-->

jesuslerma commented:
Hi I'm not shure why in the preview is redirecting all the links to production layer5.io site. But you can look at the entry here: https://5f72907fc763220009305fc0--layer5io.netlify.app/programs/hacktoberfest

Jashpatel1 commented on collections/_programs/hacktoberfest_2020.md:
```diff
@@ -0,0 +1,53 @@
+    </ol>
+    <ol>
+      Layer5 is a warm and welcoming community of open source contributors. We are continuously welcoming new members and are looking for your input. 
+      </p>
```
This closing tag is extra. Can you please remove it @jesuslerma

jesuslerma replied:
Done :)

Jashpatel1 commented on collections/_programs/hacktoberfest_2020.md:
```diff
@@ -0,0 +1,53 @@
+      <b> Layer5 has a couple of issues ready for hacktoberfest's participants </b>
+    </ol>
+    <ol>
+      We're preparing an interesting list of issues that will be good for this event. You can found that list <a href="https://github.com/search?q=org%3Alayer5io+label%3Ahacktoberfest&type=issues"> here </a>
```
@jesuslerma  Can you please update the line as "You can find the list ..." ?

jesuslerma replied:
Done :)

Jashpatel1 commented:
Nice work here @jesuslerma. Thanks for taking it up with priority.

jesuslerma commented:
You're welcome @Jashpatel1 I'm very happy to help :)

tripathyprateek reviewed (dismissed):
Looks good to me. All required changes have been done. Great job @jesuslerma

chandrashritii commented on collections/_programs/hacktoberfest_2020.md:
```diff
@@ -0,0 +1,53 @@
+    <ol>
+      Layer5 is a warm and welcoming community of open source contributors. We are continuously welcoming new members and are looking for your input. 
+
+      Please fill <a href="https://docs.google.com/forms/d/e/1FAIpQLSeQmQv6Xrnb69qGUZjHcw6i92jNDXir6WN6sRAHDNRtKUzBUg/viewform">this questionnaire </a> so our community can be prepared for a harm welcome for you and other new members.
```
```suggestion
      Help us better prepare to support you as you find your way into the world of Open Source by filling up this <a href="https://docs.google.com/forms/d/e/1FAIpQLSeQmQv6Xrnb69qGUZjHcw6i92jNDXir6WN6sRAHDNRtKUzBUg/viewform">questionnaire</a>.
```

chandrashritii commented on collections/_programs/hacktoberfest_2020.md:
```diff
@@ -0,0 +1,53 @@
+      Please fill <a href="https://docs.google.com/forms/d/e/1FAIpQLSeQmQv6Xrnb69qGUZjHcw6i92jNDXir6WN6sRAHDNRtKUzBUg/viewform">this questionnaire </a> so our community can be prepared for a harm welcome for you and other new members.
+    </ol>
+  </ul>
+Pull requests can be made in any GitHub-hosted repositories/projects.
```
```suggestion
Pull requests can be made in any of the Layer5 GitHub-hosted repositories/projects.
```

chandrashritii commented on collections/_programs/hacktoberfest_2020.md:
```diff
@@ -0,0 +1,53 @@
+    </ol>
+  </ul>
+Pull requests can be made in any GitHub-hosted repositories/projects.
+You can sign up anytime between October 1 and October 31. You can start hacking <a href="https://hacktoberfest.digitalocean.com">here</a>.
```
```suggestion
You may sign up anytime between 1st to 31st October. Begin hacking <a href="https://hacktoberfest.digitalocean.com">here</a>!
```

chandrashritii reviewed (changes requested):
Great work here. Thank you @jesuslerma A few editorial changes :)

jesuslerma commented:
Hi @chandrashritii all your requested changes have been solved. Please take a look, thanks!

leecalcote reviewed (changes requested):
<img width="1169" alt="Screen Shot 2020-09-29 at 10 51 41 AM" src="https://user-images.githubusercontent.com/7570704/94582381-c9ff1780-0241-11eb-864e-62a75141bcc9.png">

Hacktoberfest logo should be converted to grayscale to match the rest of the logos.

leecalcote commented:
Let's do what we can to avoid hyperlinks to "here". Instead hyperlink the thing itself.

leecalcote commented:
@chandrashritii where is the GSoC 2019 information on or linked to from this page? https://deploy-preview-852--layer5io.netlify.app/programs/gsoc

Nikhil-Ladha commented:
I think something has been broken here, the link to any of the programs is not working anymore. Can you just check it once?
Also, shouldn't we make the hacktoberfest icon `grayscale` too? Like the other programs.

jesuslerma commented:
Hi @Nikhil-Ladha  I think there is an error in the configuration of those program cards because each link on card is redirecting to https://layer5.io/programs/* for example if you clic on hactktoberfest card it will redirect to https://layer5.io/programs/hacktoberfest. The rest of the programs was renamed to gsod/ instead of gsod/2020 for example.

jesuslerma commented:
> @chandrashritii where is the GSoC 2019 information on or linked to from this page? https://deploy-preview-852--layer5io.netlify.app/programs/gsoc

Hi I lee I removed the GSoC 2019 card program from /programs main page because when I added the new card the layout doesn't fit well. So I asked Shriti and she approved to me to delete that program. Or maybe I didn't understand and the entry shouldn't be removed. 

Please let me know if it's ok to remove the 2019 program entry, or maybe I should add the existingt program into this entry: https://deploy-preview-852--layer5io.netlify.app/programs/gsoc

Nikhil-Ladha commented:
> Hi @Nikhil-Ladha I think there is an error in the configuration of those program cards because each link on card is redirecting to https://layer5.io/programs/* for example if you clic on hactktoberfest card it will redirect to https://layer5.io/programs/hacktoberfest. The rest of the programs was renamed to gsod/ instead of gsod/2020 for example.

Ohh, gotcha. No, the issue is with netlify deployment it happened previously too.
Can you just confirm if the pages work fine in the local deployment?

chandrashritii commented:
> 
> 
> > @chandrashritii where is the GSoC 2019 information on or linked to from this page? https://deploy-preview-852--layer5io.netlify.app/programs/gsoc
> 
> Hi I lee I removed the GSoC 2019 card program from /programs main page because when I added the new card the layout doesn't fit well. So I asked Shriti and she approved to me to delete that program. Or maybe I didn't understand and the entry shouldn't be removed.
> 
> Please let me know if it's ok to remove the 2019 program entry, or maybe I should add the existingt program into this entry: https://deploy-preview-852--layer5io.netlify.app/programs/gsoc

Yes @jesuslerma  As discussed, please add a link to the GSoC 2019 page to the GSoC (previously titled GSoC 2020) page.

jesuslerma commented:
Hi @chandrashritii and @leecalcote  I just added the prev gsoc for 2019. Also I updated the HF logo to be in grayscale, and I updated all my hyperlinks to be redirected to their real values. Please take a look.

jesuslerma commented:
@Nikhil-Ladha yes all the changes are working in my localmachine :+1:

chandrashritii commented:
> 
> 
> Hi @chandrashritii and @leecalcote I just added the prev gsoc for 2019. Also I updated the HF logo to be in grayscale, and I 
updated all my hyperlinks to be redirected to their real values. Please take a look.

Hey @jesuslerma Did you commit these changes? I don't see any new commits

jesuslerma commented:
hi @chandrashritii  sorry I forgot to push the changes. Please take a look again

tripathyprateek commented on collections/_programs/hacktoberfest_2020.md:
```diff
@@ -0,0 +1,52 @@
+    <ol>
+      Layer5 is a warm and welcoming community of open source contributors. We are continuously welcoming new members and are looking for your input. 
+
+      Help us better prepare to support you as you find your way into the world of Open Source by filling up this questionnaire: <a href="https://docs.google.com/forms/d/e/1FAIpQLSeQmQv6Xrnb69qGUZjHcw6i92jNDXir6WN6sRAHDNRtKUzBUg/viewform">https://docs.google.com/forms/d/e/1FAIpQLSeQmQv6Xrnb69qGUZjHcw6i92jNDXir6WN6sRAHDNRtKUzBUg/viewform</a>
```
Hi @jesuslerma could you just add this Hacktoberfest Meetup event link to this page as well 
`https://organize.mlh.io/participants/events/4796-getting-started-layer5`

chandrashritii commented:
LGTM

leecalcote reviewed (approved):
Thanks, @jesuslerma! 👍

leecalcote commented:
Haha! 🎈
